src/Filter/CompareFilter.php

```diff
@@ -0,0 +1,59 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Yiisoft\DataDb\Filter;
+
+use Yiisoft\Data\Reader\Filter\FilterInterface;
+
+abstract class CompareFilter implements FilterInterface
+{
+    protected string $_column;
```

Comment:
```suggestion
    protected string $column;
```

No need for underscore-prefix.